store/store.go

```diff
@@ -459,6 +460,7 @@ func New(cfg *Config, authContext auth.AuthContext) *Store {
 		store.buyURI = endpointURL(cfg.StoreBaseURL, "api/v1/snaps/purchases/buy", nil)
 		store.customersMeURI = endpointURL(cfg.StoreBaseURL, "api/v1/snaps/purchases/customers/me", nil)
 		store.sectionsURI = endpointURL(cfg.StoreBaseURL, "api/v1/snaps/sections", nil)
+		store.commandsURI = endpointURL(cfg.StoreBaseURL, "api/v1/snaps/names", nil)
```

Comment:
it's called names, but it's (going to be) a list of commands and aliases per package. I thought it best to call it for what it is, rather than for what its url says it is.

Reply:
Ironically, the only use we're making today is about names! :P

Reply:
:-) yes, for now yes